nengo/base.py

```diff
@@ -287,7 +287,7 @@ def __init__(
         self.default_dt = default_dt
         self.seed = seed
 
-    def apply(self, x, d=None, dt=None, rng=np.random, copy=True, **kwargs):
+    def apply(self, x, d=None, dt=None, rng=np.random, make_copy=True, **kwargs):
```

Comment:
I would prefer to keep this as `copy`. `np.array` uses `copy`, so there's similarity there.